src/layers/core/choropleth-layer/choropleth-layer.js

```diff
@@ -94,70 +108,107 @@ export default class ChoroplethLayer extends BaseLayer {
         drawMode: this.props.drawContour ? 'LINES' : 'TRIANGLES'
       }),
       vertexCount: 0,
-      isIndexed: true
+      isIndexed: true,
+      onBeforeRender() {
+        this.userData.oldStrokeWidth = gl.getParameter(gl.LINE_WIDTH);
+        this.program.gl.lineWidth(this.userData.strokeWidth || 1);
```

Comment:
Move the `|| 0` to where we assign strokeWidth? Principle: validate data on the edge of the component, not "inside" it.